profile/README.md

```diff
@@ -1,48 +1,40 @@
-# The Camunda Community Hub
+# Welcome to the Camunda Community Hub!
+## A project to empower our open source community extension maintainers, and encourage new contributors to get started contributing to the Camunda open source ecosystem. 
```

Comment:
```suggestion
A project to empower our open source community extension maintainers and encourage new contributors to get started contributing to the Camunda open source ecosystem. 
```

This is a bit of a nitpick/suggestion, but also important for accessibility - Only add headers to headers.